content/rest-api/applications.md

```diff
@@ -125,15 +115,148 @@ Note that the variables will need to be re-generated when moving the app to a di
 
 ### CURL example
 
-```
+```bash
 curl 'https://api.codemagic.io/apps/your-app-id/encrypt-environment-variable' \
- -H 'X-Auth-Token: your-token' \
+ -H 'X-Auth-Token: <API Token>' \
  -H 'Content-Type: application/json;charset=utf-8' \
  --data '{"value": "your value"}'
 ```
 
 ### Response
 
 ```yaml
-{"encrypted": "Encrypted(Z0FBQUFBQmZMVkhwb3Q3QlJtRlVOeVFJcEJvTTRtWnZablpqMS0xN2V6dllTell1ODZSd2FUcnNqMUlZT09QY1paV0pjbVRfUlVJeDUxRWIzX1paOEZlc1dSdi1XMXlkUFVIdjNIZ2VqcE5Ja0tpMjlPWjhlSTQ9)"}
+{
+  'encrypted': 'Encrypted(Z0FBQUFBQmZMVkhwb3Q3QlJtRlVOeVFJcEJvTTRtWnZablpqMS0xN2V6dllTell1ODZSd2FUcnNqMUlZT09QY1paV0pjbVRfUlVJeDUxRWIzX1paOEZlc1dSdi1XMXlkUFVIdjNIZ2VqcE5Ja0tpMjlPWjhlSTQ9)',
+}
 ```
+
+## Modify application variables and secrets
+
+Codemagic allows you to fetch and modify application variables and secrets using the REST API. Note that these variables are available only for applications using the `codemagic.yaml` file for configuration. They can be manually configured under **Your application > Environment variables**.

```

Comment:
```suggestion
Codemagic allows you to fetch and modify application variables and secrets using the REST API. Note that these variables are available only for the applications that are configured using the `codemagic.yaml` file. For such applications, variables and secrets are manually configured on the **Environment variables** tab in your application settings.

```

Reply:
Perhaps add a link to the env vars doc that explains how the variables are managed in the UI so there's also info about the groups